test/parse_utils_test.ts

```diff
@@ -0,0 +1,101 @@
+import { ERC20BridgeSource } from '@0x/asset-swapper';
```

Comment:
it'd be nice to add to the RFQT integration tests, to show that indicative & firm quotes return as expect when specifying `includedSources=RFQT`